src/library_html5.js

```diff
@@ -651,7 +657,7 @@ var LibraryHTML5 = {
       {{{ makeSetValue('JSEvents.wheelEvent', C_STRUCTS.EmscriptenWheelEvent.deltaY, 'wheelDeltaY', 'double') }}};
       {{{ makeSetValue('JSEvents.wheelEvent', C_STRUCTS.EmscriptenWheelEvent.deltaZ, '0 /* Not available */', 'double') }}};
       {{{ makeSetValue('JSEvents.wheelEvent', C_STRUCTS.EmscriptenWheelEvent.deltaMode, '0 /* DOM_DELTA_PIXEL */', 'i32') }}};
-      var shouldCancel = {{{ makeDynCall('iiii', 'callbackfunc') }}}( eventTypeId, JSEvents.wheelEvent, userData);
+      var shouldCancel = {{{ makeDynCall('iipp', 'callbackfunc') }}}( eventTypeId, JSEvents.wheelEvent, userData);
```

Comment:
These changes look good.  I wonder is we could figure out a test that uses these in `test_browser.py` and add `also_with_wasm64` the that test?  Then we could that this change in isolation?

Reply:
It seems like the existing tests in test-windows test-mac and test-other are failing due to some of these changes. And right, i can take a quick look and see if this can be added to a separate commit/pull request.